src/integration/helpers_test.go

```diff
@@ -4,15 +4,21 @@ import (
 	"context"
 	"crypto/tls"
 	"crypto/x509"
-	"strings"
-
+	"encoding/hex"
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	v11 "go.opentelemetry.io/proto/otlp/common/v1"
+	v1 "go.opentelemetry.io/proto/otlp/resource/v1"
+	"strings"
```

Comment:
Although it's not absolutely necessary, external libraries are typically grouped separately in the import list from standard libraries. Can you check out and apply [goimports](https://pkg.go.dev/golang.org/x/tools/cmd/goimports)?